### PR TITLE
Fix incorrect horizontal logo padding

### PR DESCRIPTION
**Pull request by SUPERCILEX**
Fixes <span><span/>#1434

Old | New
--- | ---
screenshot_20180924-162454 | screenshot_20180924-162424

### CODE REVIEW

**Comment by samtstern** [comment]
@SUPERCILEX I was just working on this, but you beat me to it!

File: auth/src/main/res/layout-land/fui_auth_method_picker_layout.xml [hunk 1]
```diff
@@ -22,6 +22,7 @@
         app:layout_constraintEnd_toStartOf="@+id/container"
         app:layout_constraintTop_toTopOf="parent"
         app:layout_constraintBottom_toTopOf="@+id/main_tos_and_pp"
+        app:layout_constraintVertical_weight="2"
```
Comment (samtstern) [inline auth/src/main/res/layout-land/fui_auth_method_picker_layout.xml]: This works, but I was also thinking what we might want is to center the logo between the top of the parent and the top of the ToS/PP.  What do you think about that?

Comment (SUPERCILEX) [reply auth/src/main/res/layout-land/fui_auth_method_picker_layout.xml]: I think it's already doing that, right? At least, that's what I'm getting from this screenshot. The issue seems to be that there's a lot more padding than we're used to.
screenshot_20180924-165834

**Comment by samtstern** [comment]
@SUPERCILEX I took a slightly different approach here:
https://github.com/firebase/FirebaseUI-Android/pull/1460

WDYT?  Not trying to step on your toes, I was already working on a fix when you sent your PR so I figured I might as well finish it and then we can decide.

**Comment by SUPERCILEX** [comment]
@samtstern well I guess it's step-over-each-others-toes week 😂. Anyway, I think we should go with your PR since it has nice sample changes, but you'll probably have to use this PR's layout changes. (I'll post the bug on your PR.)

**Pull request closed without merging**